bin/clover/src/specPipeline.ts

```diff
@@ -12,9 +12,9 @@ import {
 } from "./spec/pkgs.ts";
 
 export function pkgSpecFromCf(cfSchema: CfSchema): ExpandedPkgSpec {
-  const [aws, category, name] = cfSchema.typeName.split("::");
+  const [metaCategory, category, name] = cfSchema.typeName.split("::");
```

Comment:
I like the carefulness! Just in case we get some stuff that's got more colons, I'd add a `...more` here, and then check that more.length == 0 below.